monitoring/monitoring.go

```diff
@@ -43,25 +37,42 @@ type (
 		// this is equivalent to subtracting -1 to the current value. For Gauges,
 		// this is equivalent to setting the value to -1. For Distributions,
 		// this is equivalent to making an observation of value -1.
-		Decrement()
+		//
+		// Not used in istio, removed pending new interface.
+		//Decrement()
 
 		// Name returns the name value of a Metric.
+		// TODO: internal use only
 		Name() string
 
 		// Record makes an observation of the provided value for the given measure.
+		// Majority of Istio is setting this to an int
```

Comment:
This doesn't imply we are going to change it to `int`, right? Since all the histograms are floats (duration)

Reply:
No, I'm removing ints from this API since Istio is only using floats. Just noticing it...